samplib/ZWELNCH

```diff
@@ -34,5 +37,6 @@ RPTOPTS(ON)
 POSIX(ON)
 /*
 //STDENV   DD  *
-WORKDIR=#zwelnch
+INSTANCE_DIR=#instance_dir
+ROOT_DIR=#root_dir
```

Comment:
Any possibility to remove `ROOT_DIR` from here?

Reply:
When running as a script it would be located in ROOT_DIR, so it seems it would not be needed.
But as an STC, it seems that zowe launcher runs outside of ROOT_DIR, so it needs to know where it is.
Is that right?

Reply:
If it's STC, we should have copied ZWELNCH to loadlib. My concern is if we replace ZWESVSTC with ZWELNCH, it will be so different and how should we modify this doc: https://docs.zowe.org/stable/user-guide/configure-zowe-server.html#option-2-starting-zowe-with-a-s-tso-command.

Reply:
Yes, I see two possibilities:

1. Zowe Launcher launcher gets `ROOT_DIR` by reading and interpreting `${INSTANCE_DIR}/instance.env` file
2. Zowe Launchers executes some script inside ${INSTANCE_DIR} that could print ${ROOT_DIR} into `stdout`

Which one is better @jackjia-ibm ?